DirectEmployers/MyJobs
# [PD-2591] add datetime widget

## Overview

This ticket covers the creation of a datetime display widget similar to that found in PRM

## Testing

To test this widget, head to:
https://secure.my.jobs/prm/view/nonuseroutreach/

Select a record.
Add a partner.
Add a contact.
Check out the select boxes under the "Date & Time" field.
Update the date/time as you see fit.
Submit.

## Further work

There is still some styling to do on the form. Namely padding zeroes on the month, day, etc as well as bringing the size of the text boxes in line with the PRM tool. There will be a spin off ticket for our shiny new UI guys to make the control beautiful.
